python/python/bystro/ancestry/train.py

```diff
@@ -560,6 +560,38 @@ def get_variant(x):
     return pc_loadings
 
 
+def restrict_loadings_variants_to_vcf(
+    pc_loadings: pd.DataFrame, genos: pd.DataFrame
+) -> tuple[pd.DataFrame, pd.DataFrame, int]:
+    """Restrict variant list to overlap between gnomad loadings and reference vcf."""
+    # IGSR version of 1kgp is current reference vcf
+    var_overlap = pc_loadings.index.intersection(genos.index)
+    pc_loadings_overlap = pc_loadings[pc_loadings.index.isin(var_overlap)]
+    genos_overlap = genos[genos.index.isin(var_overlap)]
+    # Ensure that genos transpose and pc_loadings have corresponding shape and vars
+    print("genos_overlap.index", genos_overlap.index)
+    print(pc_loadings_overlap.index)
```

Comment:
this should be a logging.debug rather than a print, (I think ruff should have caught this for you?)

Reply:
Interestingly ruff didn't catch this for me and I ran ruff a few times in the process - is it a config option that maybe isn't set up? I could add it to my local dev version at the very least.